keanu-python/tests/test_sampling.py

```diff
@@ -84,8 +101,25 @@ def test_iter_returns_same_result_as_sample(algo: str) -> None:
     samples_dataframe = pd.DataFrame()
     [samples_dataframe.append(pd.DataFrame(list(next_sample.items()))) for next_sample in islice(iter_samples, draws)]
 
-    for vertex_id in samples_dataframe:
-        np.testing.assert_almost_equal(samples_dataframe[vertex_id].mean(), np.average(samples[vertex_id]))
+    for vertex in samples_dataframe:
```

Comment:
`vertex` is actually a `vertex_label`, right? (A string).

Reply:
not related to your comment, but this `samples_dataframe` is empty...

Reply:
fixed.